index.html

```diff
@@ -6,36 +6,176 @@
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <link rel="stylesheet" href="style.css">
-  <title>Livingi's Portfolio</title>
+  <title> Portfolio</title>
 </head>
 
 <body>
-  <header class="nav">
+  <header class="colour">
+    <a class="logo" href="#">My logo</a>
     <nav class="nav-bar">
-
-      <div class="logo">
-        <a href="#">My logo</a>
-      </div>
-      <div class="menu-icon"><img src="images/bar.png" alt="menu navbar"></div>
+      <a class="bars" href="#"><img src="images/menu.png" alt="menu navbar"></a>
     </nav>
   </header>
+  <main>
     <section class="first-section">
-      <div class="port-graph">
-        <h1 class="head">
+      <div class="first">
+        <h1 class="profile">
           I'm Misheck, Glad to see you!
         </h1>
-        <p class="paragraph">I’m a software developer! I can help you build a product , feature or website Look through
+        <p class="paragraph1">I’m a software developer! I can help you build a product , feature or website Look through
           some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to
           contact me.</p>
 
-        <p class="connect"> LET'S CONNECT</p>
+        <h2 class="lets-connect"> LET'S CONNECT</h2>
 
-        <ul class="social-icons" >
+        <ul class="buttons">
           <li><a href="#"><img src="./images/twitter.png" alt="social"> </a></li>
           <li><a href="#"><img src="./images/Linkedin.png" alt="social"></a></li>
           <li><a href="#"><img src="./images/medium.png" alt="social"></a></li>
           <li><a href="#"><img src="./images/github.png" alt="social"></a></li>
           <li><a href="#"><img src="./images/angellist.png" alt="social"></a></li>
         </ul>
       </div>
-    </section>
\ No newline at end of file
+    </section>
+    <section class="work-section">
+      <div class="project-card">
+
+        <img src="images/photo1.png" alt="First Card">
+
+        <h2 class="head">Tonic</h2>
+        <div class="card-detail-box">
+          <ul>
+            <li class="card-detail canopy">CANOPY</li>
+            <li class="card-detail option"><img src="images/bullets.png" alt="bullet"> Back End Dev</li>
+            <li class="card-detail option"><img src="images/bullets.png" alt="bullet"> 2015</li>
+          </ul>
+
+          <p> A daily selection of privately personalized reads; no accounts or sign-ups required. </p>
+          <div class="language1 inline">
+            <ul>
+              <li class="langu">html</li>
+              <li class="langu">css</li>
+              <li class="langu">javascript</li>
+            </ul>
+          </div>
+        </div>
+        <button type="button" class="btn-one">See project</button>
```

Comment:
Figma Design   |  Your Design
:-------------------------:|:-------------------------:
![image](https://user-images.githubusercontent.com/69638013/153702738-162e7374-6d11-4569-b60c-43946decae70.png)  |  ![hisbtn](https://user-images.githubusercontent.com/69638013/153702759-7750a3ba-9f90-4f86-8d27-b7f3341ae260.PNG)

- [ ] Kindly adjust padding for li's in your design 
- [ ] Kindly make border for buttons like the figma design
### These changes will make your design look exactly like the figma design, kindly do this for all project card  li's and  buttons  💪🏿

Reply:
Actioned.